Aztec/Classes/TextKit/TextView.swift

```diff
@@ -495,7 +495,7 @@ open class TextView: UITextView {
     // MARK: - Intercept copy paste operations
 
     open override func cut(_ sender: Any?) {
-        let data = storage.attributedSubstring(from: selectedRange).archivedData()
+        let data = try! storage.attributedSubstring(from: selectedRange).archivedData()
```

Comment:
Does that mean this line crashes if the selected attributed string contains an "attribute value" that doesn't conform to `NSCoding`? I don't really know anything about this library's implementation, not sure how likely that's going to happen...

Reply:
That's a good question, but also before the migration to Xcode 15 and newer Swift back when `archivedData` was not marked as `throws`, I'd expect the code would already crash (but maybe with an ObjC NSException rather than a Swift error being thrown)?

That being said, it would be a nice improvement to use `let data = try? …` instead and then make the subsequent call to ` storeInPasteboard(encoded: data)` conditional on `if let data`; but not sure if that fix belongs in this PR or separate.

Reply:
Also, looking at the deprecated vs new method signature for `NSKeyedArchiver.archivedData`, I'm wondering if the only case where it could `throw` is if you passed `requiresSecureCoding: true` and the objects conformed to `NSCoding`… but not `NSSecureCoding`.

But since here we use `requiresSecureCoding: false` in the implementation of `archivedData`, maybe that means we'll never throw in practice? Or if we do, that would be under the same conditions as the ones the current version of the library already does, i.e. not related to Xcode 13->Xcode15 migration?